src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -62,5 +62,12 @@ generate_projects_for_a_single_scenario <- function(forsys_input_data,
     )
   )
 
+  # Adds the input geo_wkt column to the stand output df.
+  input_stand_ids_and_geometries = select(forsys_input_data,
+                                          c(stand_id_field, geo_wkt_field)) 
```

Comment:
I don't understand what is going on here. Is this a `dplyr::select()`? Why the `c()`, is it needed? How does this add a new field, isn't the field already in the input data?

Nit: use `<-` for assignment

Reply:
This selects two columns ("stand_id" and "geo_wkt") from the input and copies them to the output (to make parsing easier).

Reply:
Ah okay, got it. In that case you don't need `c()` and could use syntax like:
```
input_stand_ids_and_geometries <- forsys_input_data %>%
    select(
        stand_id_field,
        geo_wkt_field)
```